src/SCRIPTS/BF/PAGES/pos_osd.lua

```diff
@@ -0,0 +1,169 @@
+        {["address"] = 55, ["firstVal"] = 121, ["secondVal"] = 122, ["nameCli"] = "osd_rssi_dbm_pos"},
+        {["address"] = 56, ["firstVal"] = 123, ["secondVal"] = 124, ["nameCli"] = "osd_rcchannels_pos"},
+        {["address"] = 57, ["firstVal"] = 125, ["secondVal"] = 126, ["nameCli"] = "osd_camera_frame_pos"},
+        {["address"] = 58, ["firstVal"] = 127, ["secondVal"] = 128, ["nameCli"] = "osd_efficiency_pos"},
+        {["address"] = 59, ["firstVal"] = 129, ["secondVal"] = 130, ["nameCli"] = "osd_total_flights_pos"},
+        {["address"] = 60, ["firstVal"] = 131, ["secondVal"] = 132, ["nameCli"] = "osd_up_down_reference_pos"},
+        {["address"] = 61, ["firstVal"] = 133, ["secondVal"] = 134, ["nameCli"] = "osd_link_tx_power_pos"},
+        {["address"] = 62, ["firstVal"] = 135, ["secondVal"] = 136, ["nameCli"] = "osd_wh_drawn_pos"},
+        {["address"] = 63, ["firstVal"] = 137, ["secondVal"] = 138, ["nameCli"] = "osd_aux_pos"},
+        {["address"] = 64, ["firstVal"] = 139, ["secondVal"] = 140, ["nameCli"] = "osd_ready_mode_pos"},
+        {["address"] = 65, ["firstVal"] = 141, ["secondVal"] = 142, ["nameCli"] = "osd_rsnr_pos"},
+        {["address"] = 66, ["firstVal"] = 143, ["secondVal"] = 144, ["nameCli"] = "osd_sys_goggle_voltage_pos"},
+        {["address"] = 67, ["firstVal"] = 145, ["secondVal"] = 146, ["nameCli"] = "osd_sys_vtx_voltage_pos"},
+        {["address"] = 68, ["firstVal"] = 147, ["secondVal"] = 148, ["nameCli"] = "osd_sys_bitrate_pos"},
+        {["address"] = 69, ["firstVal"] = 149, ["secondVal"] = 150, ["nameCli"] = "osd_sys_delay_pos"},
+        {["address"] = 70, ["firstVal"] = 151, ["secondVal"] = 152, ["nameCli"] = "osd_sys_distance_pos"},
+        {["address"] = 71, ["firstVal"] = 153, ["secondVal"] = 154, ["nameCli"] = "osd_sys_lq_pos"},
+        {["address"] = 72, ["firstVal"] = 155, ["secondVal"] = 156, ["nameCli"] = "osd_sys_goggle_dvr_pos"},
+        {["address"] = 73, ["firstVal"] = 157, ["secondVal"] = 158, ["nameCli"] = "osd_sys_vtx_dvr_pos"},
+        {["address"] = 74, ["firstVal"] = 159, ["secondVal"] = 160, ["nameCli"] = "osd_sys_warnings_pos"},
+        {["address"] = 75, ["firstVal"] = 161, ["secondVal"] = 162, ["nameCli"] = "osd_sys_vtx_temp_pos"},
+        {["address"] = 76, ["firstVal"] = 163, ["secondVal"] = 164, ["nameCli"] = "osd_sys_fan_speed_pos"},
+    }
+    
+if not address then
+    firstVal = items[1]["firstVal"]
+    secondVal = items[1]["secondVal"]
+    address = items[1]["address"]
+    nameCli = items[1]["nameCli"]
+end
+
+x = margin
+y = yMinLim - tableSpacing.header
+
+fields[#fields + 1] = { t = "OSD Element (select)", x = x, y = inc.y(lineSpacing), sp = x + sp, min = 0, max = (#items - 1), vals = { 1 }, upd = function(self) self.updateItems(self) end }
+fields[#fields + 1] = { t = "Screen",               x = -300, y = -300,            sp = x + sp, min = 1, max = 1, vals = { 4 } }
```

Comment:
Should be removed. Placing it outside the visible screen area will "hide" it, but the user will still be able to select it. Also we shouldn't assume that negative coordinates are handled properly on all radios. Worst case the radio could crash.